Changes

```diff
@@ -697,46 +697,46 @@ OCaml 4.07
    Xavier Leroy, Nicolas Ojeda Bar)
 
 - GPR#1661: more precise principality warning regarding record fields
-  disambiguation
+  disambiguation.
   (Thomas Refis, review by Leo White)
 
-- GPR#1687: fix bug in the printing of short functor types "(S1 -> S2) -> S3"
+- GPR#1687: fix bug in the printing of short functor types "(S1 -> S2) -> S3".
   (Pieter Goetschalckx, review by Gabriel Scherer)
 
-- GPR#1722: Scrape types in Typeopt.maybe_pointer
+- GPR#1722: Scrape types in Typeopt.maybe_pointer.
   (Leo White, review by Thomas Refis)
 
 - GPR#1755: ensure that a bigarray is never collected while reading complex
-  values (Xavier Clerc, Mark Shinwell and Leo White, report by Chris Hardin,
+  values. (Xavier Clerc, Mark Shinwell and Leo White, report by Chris Hardin,
```

Comment:
Authors should also begin on a new line

Reply:
Quick "style" question: (for my education)

After FOO#1234: Should the sentences start with a capital letter?

Reply:
I generally don’t capitalise (although I see I’ve been inconsistent with that, I notice) as to me the sentence begins, but it’s subjective. For example, it could be capital letter because the first word is a module, or it could be nice because the GPR has a title (“The Unloadable Runtime”).

I don’t think it’s worth trying to enforce a style, here - we should allow individual contributors the choice :wink: